--- rox-mixer/src/Makefile~	2005-10-19 16:36:19.000000000 +0200
+++ rox-mixer/src/Makefile	2005-10-19 16:38:15.000000000 +0200
@@ -1,7 +1,7 @@
 all:	mixer
 
 mixer:
-	gcc -o ../Linux-ix86/ROX-Mixer mixer.c -I/usr/include/gtk-2.0 -I/usr/lib/gtk-2.0/include -I/usr/include/atk-1.0 -I/usr/include/pango-1.0 -I/usr/include/freetype2 -I/usr/X11R6/include -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -Wl,--export-dynamic -lgtk-x11-2.0 -lgdk-x11-2.0 -latk-1.0 -lgdk_pixbuf-2.0 -lm -lpangoxft-1.0 -lpangox-1.0 -lpango-1.0 -lgobject-2.0 -lgmodule-2.0 -ldl -lglib-2.0
+	gcc -o ../Linux-ix86/ROX-Mixer mixer.c $(CFLAGS) `pkg-config --cflags --libs gtk+-2.0`
 
 clean:
 	rm -rf *~ *.o ../Linux-ix86/ROX-Mixer
